app/src/main/java/pl/szczodrzynski/edziennik/data/api/edziennik/github/Contributor.kt

```diff
@@ -0,0 +1,7 @@
+package pl.szczodrzynski.edziennik.data.api.edziennik.github
```

Comment:
This is not the API of an e-register. The model should be in `.models` package in the `.settings.contributors` package (or even better, as it's just a single model class, without the `.models.` suffix).